fbw-a32nx/src/systems/fmgc/src/flightplanning/A380AircraftConfig.ts

```diff
@@ -32,23 +32,23 @@ const vnavConfig: VnavConfig = {
 };
 
 const flightModelParams: FlightModelParameters = {
-  wingSpan: 262.467,
+  wingSpan: 261.65,
   wingArea: 9096,
   wingEffcyFactor: 0.7,
   requiredAccelRateKNS: 1.33,
   requiredAccelRateMS2: 0.684,
   gravityConstKNS: 19.0626,
   gravityConstMS2: 9.806665,
   machValues: [0, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95, 1.0],
-  dragCoefficientCorrections: [0, 0, 0, 0.0002, 0.0003, 0.0004, 0.0008, 0.0013, 0.002, 0.005, 0.016, 0.1],
-  speedBrakeDrag: 0.0201,
-  gearDrag: 0.00872,
+  dragCoefficientCorrections: [0, 0, 0, 0.00012, 0.00026, 0.0004, 0.00061, 0.001, 0.0018, 0.0031, 0.005, 0.0082],
+  speedBrakeDrag: 0.01154,
+  gearDrag: 0.019,
   dragPolarCoefficients: {
-    [FlapConf.CLEAN]: [0.025, -5.48e-3, -0.0192, 0.0582],
-    [FlapConf.CONF_1]: [0.0398, -0.0538, 0.1166, -0.064, 0.0303],
-    [FlapConf.CONF_2]: [0.0729, -0.0037, -0.0018, 0.0168],
-    [FlapConf.CONF_3]: [0.0902, 0.0005, -0.0056, 0.013],
-    [FlapConf.CONF_FULL]: [0.1405, -0.001, -0.0056, 0.0077],
+    [FlapConf.CLEAN]: [0.0291, 0.0171, -0.0167, 0.0236],
+    [FlapConf.CONF_1]: [0.0105, 0.0285, -0.02, 0.0305],
+    [FlapConf.CONF_2]: [0.0169, -0.0073, -0.0017, 0.1222],
```

Comment:
I just want to make sure. The polars for configurations above 2 have changed significantly, bringing down the zero lift drag but increasing the scaling coefficient by more than an order of magnitude. The coefficients are supposed to be ordered in increasing powers of Cl, i.e
`[0.0291, 0.0171, -0.0167, 0.0236]` corresponds to 
`Cd(Cl) = 0.0291 + 0.0171 * Cl - 0.0167 * Cl^2 + 0.0236 * Cl^3`.
Is this the case here?

Reply:
Oh, I thought they were in the opposite order! That's a biggie! I'll fix when I get home on Sunday unless you can do it before.